src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/project.json

```diff
@@ -1,12 +1,12 @@
 {
   "dependencies": {
-    "System.Runtime": "4.1.1-beta-24411-04",
-    "Microsoft.NETCore.Platforms": "1.0.2-beta-24411-04",
+    "System.Runtime": "4.1.1-beta-24412-02",
+    "Microsoft.NETCore.Platforms": "1.0.2-beta-24412-02",
     "System.Console": "4.0.0",
```

Comment:
Is this unchanged because we allow stable packages to remain (which is what I hope) or because System.Console isn't in your list of packages to validate?